### PR TITLE
Docs/401 create notebook with 2024q1 tech work

### PR DESCRIPTION
**Pull request by Carsopre**
## Issue addressed
Solves #<issue number>

## What has been done?
Explain how you addressed the resolution of the related issue, what choices you made and why.

## Checklist
- [x] Tests are either added or updated.
- [x] Branch is up to date with `master`.
- [x] Updated documentation if needed.

## Additional Notes (optional)
Add any additional notes or information that may be helpful.

### CODE REVIEW

File: examples/tech_meetings/20240403_using_enums_DIY.ipynb [hunk 1]
```diff
@@ -0,0 +1,349 @@
...
+    "## 2. Usage of enumerations within `ra2ce`"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Enumerations ([enum](https://docs.python.org/3/library/enum.html)) are introduced in `ra2ce` mostly to declare, and therefore distinguish, all the different properties that may influence the creation of a `network` or the run of an `analysis`.\n",
+    "\n",
+    "These enums are, in short, the different possibilities that can be fond at the `network.ini` or `analysis.ini` section properties.\n",
```
Comment (ArdtK) [inline examples/tech_meetings/20240403_using_enums_DIY.ipynb:37]:
```suggestion
    "These enums are, in short, the different possibilities that can be found at the `network.ini` or `analysis.ini` section properties.\n",
```

File: examples/tech_meetings/20240403_using_enums_DIY.ipynb [hunk 2]
```diff
@@ -0,0 +1,349 @@
...
+   "source": [
+    "In our enumeration we see two sides, the name that we will use throughout the code (left side) and its associated value (right side). In short, we are just binding said names to a __unique__ value so that python handles everything internally whilst we can write 'readable' code.\n",
+    "\n",
+    "Some pointers for creation / modification of enums here:\n",
+    "\n",
+    "- Add new properties after the latest logical value. Sometimes you may find the last two values are `BANANA = 3` and `MIX = 100`, choose then to add the next value as `PINEAPPLE = 4` instead of `PINEAPPLE = 101`.\n",
+    "- Always define the names with capital letters.\n",
+    "- Try to use simplify the names whilst still being readable.\n",
+    "- Do not modify existing values unless previously discussed with the development team.\n",
+    "- `NONE=0` and `INVALID=99` are mandatory entries when inheriting from `Ra2ceEnumBase`."
```
Comment (ArdtK) [inline examples/tech_meetings/20240403_using_enums_DIY.ipynb:171]: `NONE=0` is not mandatory and only added if the configuration attribute is optional.

**Review by ArdtK — CHANGES_REQUESTED** [review]
Few small comments

File: examples/tech_meetings/20240403_using_enums_DIY.ipynb (hunk 2 above)
Comment (Carsopre) [reply examples/tech_meetings/20240403_using_enums_DIY.ipynb:171]: You are right, are read the definition in the base class wrongly. I corrected it.

**Pull request merged**